### DISCUSSION TITLE
Is it legal to use (Individual free) in production?

### DISCUSSION THREAD

**Discussion by mhd-alsadi** [datalust/seq-tickets#1807] [General]
Is it legal to use (Individual free) in production? only one developer will access logs.
https://datalust.co/pricing

**Comment by nblumhardt** [comment]
Hi @mhd-alsadi; the Seq EULA includes all the terms for using the individual license - it's best to consult it directly, as I can't summarize effectively for you here: https://datalust.co/doc/eula-current.pdf.

Hope this helps,
Nick

**Comment by OwnageIsMagic** [comment]
https://datalust.co/privacy link in eula is broken

**Reply by nblumhardt** [reply]
Thanks for letting us know; we've fixed this now 👍
